frontend/src/components/dashboard/create/sms/SMSRecipients.tsx

```diff
@@ -127,9 +127,8 @@ const SMSRecipients = ({
       setCsvInfo((info) => ({ ...info, tempCsvFilename }))
     } catch (err) {
       setErrorMessage(err.message)
-    } finally {
-      setIsUploading(false)
     }
+    setIsUploading(false)
```

Comment:
I'm think we might still need to reset `uploading` to `false` even on error. Otherwise, when the S3 upload fails, the upload button will be stucked in a loading state even when there's an error (haven't tested this, but seems to be the case from reading the code).

Reply:
I think `setIsUploading(false)` will still be called even on errors.

Reply:
I think `setIsUploading(false)` will be called even on error. To be sure, I tested this locally by throwing an error in the `try` block. Upon error, the error message is shown and the file input is no longer in the processing state.

Reply:
Ah ok. You're right. My bad.